### ISSUE TITLE
Dashboard Settings: Unclear how to save analytics ID updates 

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#3843]
_This is somewhere between a bug and a feature - I think it's just an implementation detail that didn't get communicated/lost in the shuffle._

## Bug Description
It is not clear how to save new analytics IDs in the UI 
![Screen Shot 2020-08-12 at 11.52.06 AM.png](https://images.zenhubusercontent.com/5e9dcd9a9ff5dfdaffcdf132/2cd7c181-29c4-45ec-8694-569099fd89c0)

## Expected Behaviour
It should be clear to all users how to submit an update in this input. The same issue will exist for updating story names in the dashboard since this is a shared form component. Do we want a button of some sort or are we to save automatically when the input has lost focus? 

## Steps to Reproduce
Dashboard > Settings > update the Google Analytics Tracking ID, now you have to hit enter. 

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by samitron7** [comment]
Add a save button [here](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Stable-Release?node-id=2%3A89741)

**Comment by swissspidy** [comment]
Needs UAT
